### DISCUSSION TITLE
Table of Contents and Click to Copy Code Snippets Button for Gists Markdown

### DISCUSSION THREAD

**Discussion by ayan-b** [community/community#16153] [Other Feature Feedback, Questions, & Ideas]
Hi GitHub team,

Thanks for all the amazing work you all are doing!

Two markdown features:
- Table of contents
- Click to copy code snippets button
have landed on GitHub repos months ago but not available on GitHub Gists.

Can we have them on Github Gists too?

Thanks!

**Comment by queenofcorgis** [comment]
Thanks for sharing @ayan-b and the kind words. I've added a label to this so this ends up with the proper team. Cheers!
